libs/template/package.json

```diff
@@ -49,8 +49,11 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": ">10.1.1",
-    "@rx-angular/cdk": ">=1.0.0-beta.2"
+    "@angular/core": ">=8.0.0",
+    "@rx-angular/cdk": ">=1.0.0-beta.2",
+    "@angular/cdk": ">=8.0.0",
```

Comment:
this is true, but I would rather remove the dependency to angular/cdk. We only use it once in the `rxIf`. Can u remove that dependency and adjust the package.json?

Reply:
We can't remove `@angular/cdk` because ng-packagr will put it in the distribution package.json at build time with a wrong version range, I think the best here is to use the optional peer dependency flag.

Reply:
sorry, my comment led to a misunderstanding. the plan is to remove the usage of `angular/cdk` in rxIf, so that we can remove the dependency from the package.json as well :) it's just the one function which we can replace easily